### PR TITLE
chore(): add ionic issue bot

### PR DESCRIPTION
**Pull request by sean-perkins**
Adds the Ionitron (ionic issue bot) to the repository to aid with repository management.

### CODE REVIEW

**Review by rwaskiewicz — CHANGES_REQUESTED** [review]
Two small asks before we land this:

1. We should document (internally) how we expect the Framework + Stencil team should triage issues - what labels to apply/when, etc. I'd be willing to bet there's been some divergence in our respective repository's processes.  That's fine (no complaints there!), but I want to make sure there's a reference point for both teams to understand expectations around handling issues in the repo
2. When we finalize the labels/process described here, we'll need to make sure these labels get added to https://github.com/ionic-team/stencil-ds-output-targets/labels

This looks great though! Thanks sooo much for doing this 🙏

**Comment by sean-perkins** [comment]
Stale section has been removed. I also dropped a notion doc in our internal channel for the initial process.

**Review by amandaesmith3 — APPROVED** [review]

File: .github/ionic-issue-bot.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        This issue has been labeled as `help wanted`. This label is added to issues
+        that we believe would be good for contributors.
+
+
+        If you'd like to work on this issue, please comment here letting us know that
+        you would like to submit a pull request for it. This helps us to keep track of
+        the pull request and make sure there isn't duplicated effort.
+
+        Thank you!
+    - label: "ionitron: needs reproduction"
```
Comment (rwaskiewicz) [inline .github/ionic-issue-bot.yml:19]: Does this label / action conflict with the one below? https://github.com/ionic-team/stencil-ds-output-targets/pull/240/files#diff-5ab3559a040d16f7b7db847060a448dddde18e8320e13b8d5737c31672708375R66

The message associated with this action seems to imply the issue will stay open (so long as they provide a repro), whereas the one below closes the issue. Am I reading this right?

**Comment by rwaskiewicz** [comment]
@sean-perkins I think we're in a good place to add the labels in the PR to https://github.com/ionic-team/stencil-ds-output-targets/labels. Thoughts?

**Comment by sean-perkins** [comment]
Yep, I can add those labels now 👍

File: .github/ionic-issue-bot.yml (hunk 1 above)
Comment (rwaskiewicz) [reply .github/ionic-issue-bot.yml:19]: 👍 Makes sense to me - thanks! (serious brain fog day 🙃)

**Review by rwaskiewicz — APPROVED** [review]
Approved, Sean's adding the labels now

**Pull request merged**